docs/proposals/multiple-sources-for-applications.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Neat-enhancement-idea
+authors:
+  - "@ishitasequeira" # Authors' github accounts here.
+sponsors:
+  - TBD        # List all intereste parties here.
+reviewers:
+  - "@jannfis"
+  - TBD
+approvers:
+  - "@jannfis"
+  - TBD
+
+creation-date: 2022-01-28
+last-updated: 2022-01-28
+---
+
+# Multiple Sources for application
+
+Support more than one source for creating an Application.
+
+Related Issues: 
+* https://github.com/argoproj/argo-cd/issues/677
+* https://github.com/argoproj/argo-cd/issues/2789
+
+## Open Questions
+* Adding external sources to the Application reource would add additional latencies for creation/reconcilation process. Should we add it to the doc in Risks?
+* 
+
+## Summary
+
+Currently, Argo CD supports Applications with only a single ApplicationSource. In certain scenarios, it would be useful to support more than one source for the application. For example, consider a need to create multiple applications with the same manifests. Today we would have to copy manifest files from one application to another.
```

Comment:
For example, an independent support of the Helm charts and the values files. This opens a door to the following highly requested scenarios: native deployments of the same (possibly external) Helm chart with different values files or an independent migration to a newer Helm chart version for the same applications installed in Test and Production environments.

Reply:
I also found that statement about "create multiple applications with the same manifests" is somewhat misleading because the proposal is about supporting multiple sources for a single Application.  Please rephrase it. I think you mean "create multiple deployments of the same application and manifests but manifests can come from different sources".  Perhaps, you can cite the helm charts example above.